### ISSUE TITLE
[PROXY][1.0.3] "Proxy is not correctly started"

### ISSUE THREAD

**Issue by oebeledrijfhout** [dut-iptv/dut-iptv#15]
After a while, usually a few hours, I observe the following message from the plugin: "Proxy is correctly started, this often happens after first installing or updating, please restart Kodi and try again." And the NLZiet plugin stops working, I can't watch live tv nor replay tv until after I restart the device (platform is LibreELEC 9.2.6).

In `kodi.log` the following error is logged:

```
2021-02-08 21:30:33.665 T:140368255383296   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.AttributeError'>
                                            Error Contents: 'NoneType' object has no attribute 'serve_forever'
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 863, in <module>
                                                main()
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 393, in main
                                                service.reloadHTTPServer()
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 275, in reloadHTTPServer
                                                self.startHTTPServer()
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 293, in startHTTPServer
                                                threadStarting = threading.Thread(target=self.HTTPServer.serve_forever)
                                            AttributeError: 'NoneType' object has no attribute 'serve_forever'
                                            -->End of Python script error report<--
```

Other plugins I'm using: Emby and Librespot.

**Comment by dut-iptv** [comment]
Proxy is automatically restarted every (few?) hours because of stale connections. Error probably occurs around a restart, will look into it and consider a fix although officially leia development was halted

**Comment by dut-iptv** [comment]
I have uploaded a possible fix for proxy 1.0.3, I was however not ably to reproduce your issue on Kodi Matrix.

This will be the only attempt to fix this issue on Leia.
If this does not resolve the issue upgrade to Matrix and create a new issue if you still encounter this issue at that time.

**Comment by oebeledrijfhout** [comment]
I would, but I can't update to Matrix yet as some of the plugins I'm using don't work with it currently. 

Version of Proxy is now 1.0.3, unfortunately I still see the same error:

```
2021-02-10 19:15:37.949 T:139644637255424   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.AttributeError'>
                                            Error Contents: 'NoneType' object has no attribute 'serve_forever'
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 863, in <module>
                                                main()
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 393, in main
                                                service.reloadHTTPServer()
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 275, in reloadHTTPServer
                                                self.startHTTPServer()
                                              File "/storage/.kodi/addons/service.dutiptv.proxy/service.py", line 293, in startHTTPServer
                                                threadStarting = threading.Thread(target=self.HTTPServer.serve_forever)
                                            AttributeError: 'NoneType' object has no attribute 'serve_forever'
                                            -->End of Python script error report<--
```

Hopefully I can update to Matrix soon.
